### DISCUSSION TITLE
trivy_0.52.1_Linux-64bit.tar.gz - The specified blob does not exist

### DISCUSSION THREAD

**Discussion by Kerwood** [aquasecurity/trivy#6891] [Bugs]
### Description

When trying to download `trivy_0.52.1_Linux-64bit.tar.gz` from the release page I get an error.
```
<Error>
  <Code>BlobNotFound</Code>
  <Message>The specified blob does not exist. RequestId:12b3b3b6-d01e-0064-2222-bb10cd000000 Time:2024-06-10T10:41:09.8253685Z</Message>
</Error>
```
I am able to download other files from the same release. 

### Desired Behavior

It would be very desirisble to be able to download the file.

### Actual Behavior

Error appears.

### Reproduction Steps

```bash
Just click the `trivy_0.52.1_Linux-64bit.tar.gz ` link on the release page.
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
none
```

### Operating System

github.com

### Version

```bash
none
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by DmitriyLewen** [comment]
duplicate https://github.com/aquasecurity/trivy/discussions/6890#discussioncomment-9726060
